go-controller/pkg/node/management-port_linux.go

```diff
@@ -135,7 +137,7 @@ func tearDownManagementPortConfig(mpcfg *managementPortConfig) error {
 	}
 
 	if mpcfg.ipv4 != nil {
-		if err := util.LinkRoutesDel(mpcfg.link, mpcfg.ipv4.allSubnets); err != nil {
+		if err := util.LinkRoutesDel(mpcfg.link, nil); err != nil {
```

Comment:
we can move this out, just after LikAddrFlush in L135,  so we don't do it twice both for ipv4 and ipv6

Reply:
good point. will do.